### ISSUE TITLE
A11y_SQLdatabase projects extension_Publish_New SQL Server_Voiceover : When the "I accept Microsoft SQL Server License Agreement" tick-box is not selected/ticked then Voiceover is announcing it as "Invalid data".

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/azuredatastudio#22221]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev3; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #WCAG1.3.1; #MAC; #AINotInScope; #Info and Relationships; #Voiceover; #Element:Checkbox;

### Actual Result:​
When the "I accept Microsoft SQL agreement" tick-box is not selected/ticked then Voiceover is announcing it as "Invalid data".

### Expected Result:​
Voiceover should not announce the non-selected checkbox as invalid, Invalid data should only announced when there is any wrong value is entered for the control.

### Environment Details:​
Application Name: SQL Azure Data Studio
Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

### Repro Steps:

1. Start Voiceover with using shortcut key Command+F5.
2. Launch Azure Data Studio
3. Tab till extension and search for “SQL Database Projects” and install it.
4. Tab till database project and press enter.
5. “Database Project” panel will open.
6. Tab till existing project name and press context menu button(right click).
7. Tab till "Publish" menu-item in list and press enter.
8. Tab till "Publish target" group and select "Publish to new azure SQL local server" radio button.
9. Tab till "I accept Microsoft SQL Server License Agreement" tick box.
10. Verify when the "I accept Microsoft SQL agreement" tick-box is not selected/ticked then Voiceover is announcing it as "Invalid data" or not.
### User Impact:

Voiceover users will get confused regarding the control input type as without entering any value it is announcing the field as invalid for checkbox.

### Attachment:
![publish to new sql - i agree checkbox (1)](https://user-images.githubusercontent.com/91586101/223651576-154b57ac-ce1b-4c75-916d-c8e1c0b85aaa.gif)
